mne/viz/epochs.py

```diff
@@ -116,14 +116,21 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
             ax3 = axes[-1]
     evoked = epochs.average(picks)
     data = epochs.get_data()[:, picks, :]
+    n_epochs = len(data)
+    data = np.swapaxes(data, 0, 1)
```

Comment:
I could do the `np.swapaxes` in one line with `epochs.get_data()` and then set `n_epochs = data.shape[1]` if you prefer.